callback.go

```diff
@@ -26,6 +26,15 @@ type Callback struct {
 	// a bad client can send arbitrary data in this field.
 	Data string `json:"data"`
 
+	// Global identifier, uniquely corresponding to the chat to
```

Comment:
Please fix the documentation:

```suggestion
	// ChatInstance is a global identifier, uniquely corresponding to the chat to
```

Reply:
Done